src/containers/NewInvoicePage.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Immutable from 'immutable';
+import { connect } from 'react-redux';
+import { createInvoice } from '../thunks';
+import  CreateInvoiceForm from '../components/CreateInvoiceForm';
+
+function NewInvoicePage({clients, users, onCreateClick}) {
+  return (
+    <CreateInvoiceForm onSubmit={onCreateClick}
+                         clients={clients}
+                         users={users}/>
+  );
+}
+
+function clientFromProject(project) {
+  const name = project.get('name');
+  return name.split('|', 2)[0];
```

Comment:
@ssrihari This is a time-tracker specific convention. Client names and project names are captured together now. Hence you need this munging. Eventually we need to move away to a proper/better (nested) data structure to capture Client->Project(-> Task) information.